source/guides/applications/initializers.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Lotus - Guides - Applications Initializers
+---
+
+# Initializers
+
+Each single Lotus application within a project can have one or more custom initializers.
+They are **optional**.
+
+<p class="notice">
+  Initializers are optional
+</p>
+
+They are run **after** the dependencies, the framework and the application code are loaded, but **before** the server or the console are started.
+
+<p class="convention">
+  For a given application named <code>Web</code>, they MUST be placed under <code>apps/web/config/initializers</code>.
+</p>
+
+Their execution order is alphabetical.
```

Comment:
@joneslee85 Alphabetic order derives from `Dir.glob`, which is a Ruby way to list files. Here we're just remembering it again.

If we suggest these tricks we would introduce another problem: versioning in a large dev team and Git conflicts. This is a similar problem that Rails had back in the day with numeric migrations.

This initializers feature is well known because of Rails, devs can figure out what's works best for their team.